prow/cmd/sidecar/main.go

```diff
@@ -18,28 +18,34 @@ package main
 
 import (
 	"context"
+	"io"
+	"os"
 
 	"github.com/sirupsen/logrus"
 
-	"k8s.io/test-infra/prow/logrusutil"
 	"k8s.io/test-infra/prow/pod-utils/options"
 	"k8s.io/test-infra/prow/sidecar"
 )
 
 func main() {
-	logrusutil.ComponentInit()
-	logrus.SetLevel(logrus.DebugLevel)
-
+	logFile, err := sidecar.LogSetup()
+	if err != nil {
+		logrus.Errorf("Unable to set up log file")
```

Comment:
We should probably use `logrus.Fatal` here since we assume `logFile` is valid throughout the rest and a violation of that assumption could cause a panic.
Also we don't need the `f` suffix since this isn't using a format string.

Reply:
Please also use `.WithError(err)` to include the actual error message in the log.

Reply:
if err != nil {
		logrus.Fatal("Unable to set up log file")
		logrus.WithError(err).Error("%s")
	}
like this?

Reply:
Not quite. Fatal exits the process so we wouldn't reach that error log. This is what I mean:
```suggestion
		logrus.WithError(err).Fatal("Unable to set up log file")
```